crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -28,9 +28,12 @@ def initialize(backup)
       end
 
       def restore_background
+        Rails.logger.debug("Starting restore in the background")
         @thread = Thread.new do
+          Rails.logger.debug("Entering thread: #{Thread.current.inspect}")
```

Comment:
Well, that's three log lines -- I'm not sure we need that much. I think just one inside the thread saying "Starting restore in background thread" is enough. And no need to give info about the thread, the prefix in the log gives that already.

The important bit to me is to have one line at the beginning of the thread so that we can identify in the log which thread is responsible for restoring.